Automattic/wp-calypso
# Copy Site: Add copy site to General Settings Site Tools

#### Proposed Changes

Add copy site to site tools

#### Testing Instructions



1. Navigate to `Settings -> General -> Site Tools`
2. Ensure you have a site with a `Business` plan
3. Make sure that you see the "Site tools" section

<img width="841" alt="image" src="https://user-images.githubusercontent.com/36432/214060577-402af5ea-b62c-4d66-84c6-1dfebaa97055.png">

4. Press the copy site, and go through the steps. Ensure that your site is copied successfully.
5. Make sure that you can see the analytics tracking when you click the copy site button

<img width="1626" alt="image" src="https://user-images.githubusercontent.com/36432/214062039-72203e5a-1934-4a90-bd43-fb38e8bea31a.png">

#### Pre-merge Checklist





Fixes https://github.com/Automattic/dotcom-forge/issues/1508
